_pytest/tmpdir.py

```diff
@@ -54,9 +54,8 @@ def getbasetemp(self):
             else:
                 # use a sub-directory in the temproot to speed-up
                 # make_numbered_dir() call
-                import getpass
                 temproot = py.path.local.get_temproot()
-                rootdir = temproot.join('pytest-of-%s' % getpass.getuser())
+                rootdir = temproot.join('pytest-of-%s' % get_user())
```

Comment:
> If you have a Jenkins machine that executes multiple tox runs is this not bound to lead to bad interactions?

Hmm I don't think so, because `py.path.local.make_numbered_dir` is atomic, and will correctly create a separate directory for each invocation... `pytest-tox` will only be used when running py.test from inside `tox` without passing `%USERNAME` on Windows.

> What about simply not creating a subdir but using the "slow" make_numbered_dir on the temproot?

That's how the code was written originally, but it was changed as an optimization. From the CHANGELOG:

> - optimized tmpdir fixture initialization, which should make test sessions
  faster (specially when using pytest-xdist). The only visible effect
  is that now pytest uses a subdirectory in the $TEMP directory for all
  directories created by this fixture (defaults to $TEMP/pytest-$USER).